### PR TITLE
add some worship and hymns

### PR DESCRIPTION
**Pull request by knight3001**
add worship: 08_祢眼目必看顧這地, 08_所有的榮耀歸於祢, hymns: 066_榮耀歸主名, 416_十架道路

### CODE REVIEW

File: OHPGen/Core/Assets/hymns/066_榮耀歸主名.ini [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[conf]
+id=66
+name=榮耀歸主名
+count=3
+
+[0]
+verse=在十字架上 我救主捨命\n靠主十架 我罪得洗淨\n除我重擔 安慰我傷心\n榮耀歸主名
```
Comment (eFiniLan) [inline OHPGen/Core/Assets/hymns/066_榮耀歸主名.ini]: 全型空白「　」

File: OHPGen/Core/Assets/hymns/416_十架道路.ini [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+verse=你要變節或忠心到死\n讓一切完全損失\n直等到永活主的豐盛生命\n天天充滿在你靈
+chorus=這是十架道路   你願否走這個\n你曾否背十架為你主\n你這奉獻一切給神的人\n對神   你是否全貞
+
+[3]
+verse=神的救法是死裡得生\n你與主合為一人\n在十字架上因信與祂同釘\n從你身活出祂命
+chorus=這是十架道路   你願否走這個\n你曾否背十架為你主\n你這奉獻一切給神的人\n對神   你是否全貞
+
+[4]
+verse=我們的得失並不要緊\n神的旨意當留心\n我們若將萬事都看如糞土\n主才不會攔阻
+chorus=這是十架道路   你願否走這個\n你曾否背十架為你主\n你這奉獻一切給神的人\n對神   你是否全貞
```
Comment (eFiniLan) [inline OHPGen/Core/Assets/hymns/416_十架道路.ini]: 一個全型空白

File: OHPGen/Core/Assets/worship/08_祢眼目必看顧這地.ini [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[conf]
+name=祢眼目必看顧這地
+count=3
+
+[lyric]
+0=大聲敬拜全心呼喊\n全能的神願祢降臨\n\n慈愛雙手大能膀臂\n直到如今仍神蹟
+1=祢眼目 必看顧這地 眷顧祢的兒女\n祢必照 所應許的成就\n祢眼目 必看顧這地 聆聽我們聲音\n禱告盼望 榮耀復興降臨
```
Comment (eFiniLan) [inline OHPGen/Core/Assets/worship/08_祢眼目必看顧這地.ini]: 全型空白

**Comment by eFiniLan** [comment]
白空部分換全型空白，然後 pr 到 development (我改好了) 💯

**Comment by knight3001** [comment]
全型空白是啥？切换成全角再打空白？这。。复制过来的都不是，你们都是手动改的？

**Comment by knight3001** [comment]
anyway,我下周再做时改了-_-

**Comment by knight3001** [comment]
> 1 space instead of 2? don't see a point of having 2 lol

original copy have 3.....maybe for beauty?

**Review by eFiniLan — COMMENTED** [review]
> > 1 space instead of 2? don't see a point of having 2 lol
> 
> original copy have 3.....maybe for beauty?

yeah, try to align with the previous line I guess, so just change it to 1 space, we don't need 2.

**Pull request merged**